### PR TITLE
INTYGFV-15015: Implemented category aktivitetsbegransning with questions.

### PR DESCRIPTION
**Pull request by ChrilleSigerhed**

### CODE REVIEW

File: fk/luae_na/src/main/java/se/inera/intyg/common/luae_na/v1/model/converter/certificate/question/QuestionAktivitetsbegransningar.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+import se.inera.intyg.common.support.facade.util.ValueToolkit;
+
+public class QuestionAktivitetsbegransningar {
+
+    private static final short TEXT_LIMIT = 4000;
+
+    public static CertificateDataElement toCertificate(String textValue, int index, CertificateTextProvider textProvider) {
+        return CertificateDataElement.builder()
+            .id(AKTIVITETSBEGRANSNING_DELSVAR_ID_17)
+            .parent(AKTIVITETSBEGRANSNING_CATEGORY_ID)
```
Comment (mww-sopra-steria) [inline fk/luae_na/src/main/java/se/inera/intyg/common/luae_na/v1/model/converter/certificate/question/QuestionAktivitetsbegransningar.java:45]: Shouldn't this have the "Header"-question as parent?

**Review by mww-sopra-steria — CHANGES_REQUESTED** [review]
Looks good! Just a small question/clarification to look at.

File: fk/luae_na/src/main/java/se/inera/intyg/common/luae_na/v1/model/converter/certificate/question/QuestionAktivitetsbegransningar.java (hunk 1 above)
Comment (ChrilleSigerhed) [reply fk/luae_na/src/main/java/se/inera/intyg/common/luae_na/v1/model/converter/certificate/question/QuestionAktivitetsbegransningar.java:45]: My last two commits highlights my own confusion regardning this :)

When I used the header as parent, the distance between the header & the question increased. To get closer to how it looks in the angular client, I therefore changed the parent back to category. Im not sure if this should be handled in frontend and we keep the same structure as we did for Dodsplatskommun in Dodsbevis.

**Review by mww-sopra-steria — APPROVED** [review]
OK! Understand, will approve based on your answer.

**Pull request merged**